solidportal/panel/src/components/UserInputView.js

```diff
@@ -387,6 +388,7 @@ const UserInputView = ({   showView,
                     setGetAutoGenStatusCall(true)
                 } else if (response.data.status === 2) {
                     setIsAutoGenNewSession(true)
+                    getAutoGenResult.current = ""
```

Comment:
This is invalid graph response code, not the new session create, we shoud clean up the output editor in status == 1, and then populate the new response content.